apps/src/code-studio/videos.js

```diff
@@ -423,15 +461,24 @@ function addFallbackVideoPlayer(videoInfo, playerWidth, playerHeight) {
         }
       });
 
+      function onResize() {
```

Comment:
Similar question about this `onResize`.  In general, I'm not a big fan of having the same function name multiple times in one file.